pandas/parser.pyx

```diff
@@ -1239,6 +1239,10 @@ class CParserError(ValueError):
     pass
 
 
+class EmptyDataError(ValueError):
+    pass
```

Comment:
Well if we're moving `CParserError` and `EmptyDataError` from `parser.pyx` to `io/common.py`, any thoughts about `OverflowError`?